Release_ISOs

```diff
@@ -13,13 +13,20 @@ if [ "$ROS_ARCH" = "" ]; then
 	exit 1
 fi
 
+if [ "$ROS_ARCH" = "i386" ]; then
+	ROS_PRETTY_ARCH="x86"
+else
+	ROS_PRETTY_ARCH="$ROS_ARCH"
+fi
+
+
 # Constants
 ROOTDIR="$PWD"
 OUTPUTDIR="output-MinGW-i386"
-BOOTCDISO="ReactOS-${version}-${ROS_ARCH}.iso"
-BOOTCDZIP="ReactOS-${version}-${ROS_ARCH}-iso.zip"
-LIVECDISO="ReactOS-${version}-${ROS_ARCH}-live.iso"
-LIVECDZIP="ReactOS-${version}-${ROS_ARCH}-live.zip"
+BOOTCDISO="ReactOS-${version}-${ROS_PRETTY_ARCH}.iso"
+BOOTCDZIP="ReactOS-${version}-${ROS_PRETTY_ARCH}-iso.zip"
+LIVECDISO="ReactOS-${version}-${ROS_PRETTY_ARCH}-live.iso"
+LIVECDZIP="ReactOS-${version}-${ROS_PRETTY_ARCH}-live.zip"
```

Comment:
Ok, the iso and zip name generation works like a charm now after this PR. Problem solved.
![image](https://github.com/user-attachments/assets/b52e3d37-d672-4294-b5c5-c7aeee58d4ca)